next-tavla/src/Admin/scenarios/Boards/components/FilterButton/styles.module.css

```diff
@@ -0,0 +1,13 @@
+.tagsFilterPopover {
+    max-width: 20rem;
+}
+
+.buttonWrapper {
+    position: relative;
+}
```

Comment:
Dette kan vi lage til en global klasse

Reply:
Var inne på tanken, men bruker bare den verdien ett sted og må assigne klassen for å ha en selector på span-elementet inni. Kan godt kjøre en kombo?

Reply:
Blir i såfall en ny variant av classNames(classes.buttonWrapper, 'positionRelative'). Vet ikke om det er det vi vil